python/katana/cpp/libgalois/graphs/Graph.pxd

```diff
@@ -132,11 +132,14 @@ cdef extern from "katana/Graph.h" namespace "katana" nogil:
 
     cppclass _PropertyGraph "katana::PropertyGraph":
         PropertyGraph()
-        PropertyGraph(GraphTopology&&)
+        # PropertyGraph(GraphTopology&&)
```

Comment:
@amberhassaan this seems like a mis-commit?

Reply:
no, I intend to remove it eventually. It's replaced by the `Make` function because this constructor no longer exists in C++.